protobuf/leaderboardstat/leaderboardstat.proto

```
@@ -26,6 +26,25 @@ message ContributorStatRequest{
+  repeated PublicLeaderboardRow rows = 2;
+}
+
+message PublicLeaderboardRow {
+  uint64 user_id = 1;
+  uint64 rank = 2;
+  double score = 3;
+}
+
 service LeaderboardStatService{
   rpc GetContributorStats(ContributorStatRequest) returns (ContributorStatResponse);
+  rpc GrtPublicLeaderboard(GetPublicLeaderboardRequest) returns (GetPublicLeaderboardResponse);
```

Buf (1.61.0)

RPC request type "GetPublicLeaderboardRequest" should be named "GrtPublicLeaderboardRequest" or "LeaderboardStatServiceGrtPublicLeaderboardRequest".

(RPC_REQUEST_STANDARD_NAME)